data/Britton_1994/metadata.yml

```diff
@@ -0,0 +1,229 @@
+      are located at altitudes between 1000 and 1325 m on the New England plateau
+      in northern New South Wales
+contexts: .na
+traits:
+- var_in: germination_glasshouse
+  unit_in: .na
+  trait_name: recruitment_time
+  entity_type: population
+  value_type: mode
+  basis_of_value: measurement
+  replicates: 30
+  basis_of_record: lab
+  methods: Four germination trials were run in seasonal succession, commencing in
+    September 1992 (spring 1) and ending in August 1993 (winter). A fifth trial (spring
+    2) was conducted in September 1993 in order to assess whether dry storage of the
+    sediments over a 12-month period had any influence on germination response. Dry
+    storage may increase the sensitivity of seeds to  emperature and hence increase
+    the likelihood of germination (Thompson and Grime 1983; Probert et al. 1987).
+    In each trial, 125-mL subsamples of dried sediment were placed in clear plastic
+    trays (16 X 10 X 5 cm), watered with tap water, and maintained in a waterlogged
+    condition for a period of eight weeks. Thirty trays (six sites x five replicates)
+    were placed at random within a shallow, Polythene-lined tank situated outdoors
+    in the grounds of the University, and thirty (six sites X five replicates) were
+    placed within a tank inside a glasshouse. The trays were perforated at their bases
+    to allow movement of water from the tanks into the trays. Artificial lighting
+    or heating was not used in the glasshouse, although a cooling system, which came
+    into operation at around 27OC, maintained temperatures in the glasshouse below
+    30°C for most of the year. At the end of the eight-week wetting period, individuals
+    that had germinated were identified and counted. The sediments were then discarded.
+    Maximum-minimum thermometers were used to measure air temperatures and water temperatures
+    within the tanks, both in the glasshouse and outside. Readings were taken at approximately
+    three-day intervals throughout each germination trial. Light measurements were
+    recorded at midday at the same intervals with the aid of a Li-Cor (LI-185B) photometer,
+    which measures radiation in the 400 to 700 nm waveband. Data obtained in the first
+    four trials (spring 1, summer, autumn and winter) on species richness, total numbers
+    and abundance of dominant species at each site were compared between seasons and
+    between treatments (outdoor v. glasshouse) by two-way analysis of variance (ANOVA).
+    Comparisons of numbers of species, total numbers and the abundance of dominant
+    species at each site in the two spring trials (spring 1 and spring 2) were carried
+    out with Student's t-test.
+- var_in: germination_outdoors
+  unit_in: .na
+  trait_name: recruitment_time
+  entity_type: population
+  value_type: mode
+  basis_of_value: measurement
+  replicates: 30
+  basis_of_record: field_experiment
```

Comment:
Is this technically `captive_cultivated` if it was on the grounds of the university? Also does that mean that the site should be the University of New England and there should be a context variable for seed provenance (wetlands in northern New South Wales)?

Reply:
Captive cultivated is meant for an arboretum or common garden experiment - I think for seed germination it really is just an experiment. But I did switch the seed provenance to a context property and re-did location.